packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -368,174 +416,250 @@ const SchedulerForm: FC<Props> = ({
-                                                }
-                                            />
-                                        }
-                                        onClick={() =>
-                                            setShowFormatting((old) => !old)
-                                        }
-                                    >
-                                        Formatting options
-                                    </Button>
-                                    <Collapse in={showFormatting} pl="md">
-                                        <Group align="start" spacing="xxl">
-                                            <Radio.Group
-                                                label="Values"
-                                                {...form.getInputProps(
-                                                    'options.formatted',
-                                                )}
-                                            >
-                                                <Stack spacing="xxs" pt="xs">
-                                                    <Radio
-                                                        label="Formatted"
-                                                        value={Values.FORMATTED}
-                                                    />
-                                                    <Radio
-                                                        label="Raw"
-                                                        value={Values.RAW}
-                                                    />
-                                                </Stack>
-                                            </Radio.Group>
-                                            <Stack spacing="xs">
+                                <Group noWrap grow>
+                                    <Select
+                                        required
+                                        label="Condition"
+                                        data={thresholdOperatorOptions}
+                                        {...form.getInputProps(
+                                            `thresholds.0.operator`,
+                                        )}
+                                    />
+                                    <NumberInput
+                                        label="Threshold"
+                                        {...form.getInputProps(
+                                            `thresholds.0.value`,
+                                        )}
+                                    />
+                                </Group>
+                            </Stack>
+                        )}
+                        {isAlert ? (
+                            <Input.Wrapper label="Frequency">
+                                <Text color="gray" mt={8}>
+                                    Alerts will be processed at 10am daily
```

Comment:
perhaps add `UTC` ?

Reply:
😅 Ok. Is that right? When people pick a time is it in UTC?

Reply:
I put it in there. If it's wrong, we are going to change this soon anyway.